### PR TITLE
fix(VRipple): use $ripple-animation-visible-opacity variable

### LINKED ISSUE #11505: [Bug Report] $ripple-animation-visible-opacity ignored
### Environment
**Vuetify Version:** 2.2.30
**Vue Version:** 2.6.11
**Browsers:** Chrome 83.0.4103.61
**OS:** Mac OS 10.14.5

### Steps to reproduce
Adding "$ripple-animation-visible-opacity: 0.05" to the variable list

### Expected Behavior
To have a ripple with a visible opacity of 5% instead of the 15% default

My current workaround is to do:
```scss
.v-ripple__container {
  opacity: 0.5;
}
```

### Actual Behavior
Changing this variable doesn't affect the visibility of the ripple

### Reproduction Link
<a href="https://codepen.io/melalj/pen/pojmYoL" target="_blank">https://codepen.io/melalj/pen/pojmYoL</a>

### Other comments
On the source code there's no mention of the usage of the variable: $ripple-animation-visible-opacity ( https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/directives/ripple/VRipple.sass)

Am I missing something?

### PR DESCRIPTION
**Pull request by KareemDa**
## Description

fixes #11505 
- removed variables from `packages/vuetify/src/directives/ripple/_variables.scss` as they are already defined in `packages\vuetify\src\styles\settings\_variables.scss`
- removed opactiy functionality from `Vripple index.ts` as it overrides the opactiy in sass file 
- added `opacity` variable, functionality into sass file

## Markup:

<details>

```vue
<template>
  <v-app id="inspire">
    <div v-ripple class="text-center elevation-2 pa-12 text-h5">
      HTML element with v-ripple
    </div>
  </v-app>
</template>

<script>
export default {};
</script>
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/directives/ripple/_variables.scss [hunk 1]
```diff
@@ -1,6 +1 @@
 @import '../../styles/styles.sass';
-
-$ripple-animation-transition-in: transform .25s map-get($transition, 'fast-in-slow-out'),
-                                 opacity .1s map-get($transition, 'fast-in-slow-out') !default;
-$ripple-animation-transition-out: opacity .3s map-get($transition, 'fast-in-slow-out') !default;
-$ripple-animation-visible-opacity: .15 !default;
```
Comment (KaelWD) [inline packages/vuetify/src/directives/ripple/_variables.scss:6]: Prever this file over the global one if the variables aren't used anywhere else.

File: packages/vuetify/src/directives/ripple/index.ts [hunk 2]
```diff
@@ -109,14 +105,12 @@ const ripples = {
...
     animation.classList.add('v-ripple__animation--visible')
     transform(animation, `translate(${x}, ${y}) scale3d(${scale},${scale},${scale})`)
-    opacity(animation, 0)
     animation.dataset.activated = String(performance.now())
 
     setTimeout(() => {
       animation.classList.remove('v-ripple__animation--enter')
       animation.classList.add('v-ripple__animation--in')
       transform(animation, `translate(${centerX}, ${centerY}) scale3d(1,1,1)`)
-      opacity(animation, 0.25)
```
Comment (KaelWD) [inline packages/vuetify/src/directives/ripple/index.ts:119]: Does this change the appearance of ripples? The stylesheet uses 0.15

Comment (KareemDa) [reply packages/vuetify/src/directives/ripple/index.ts:119]: Yes, it does. it overrides sass values.

File: packages/vuetify/src/directives/ripple/_variables.scss (hunk 1 above)
Comment (KareemDa) [reply packages/vuetify/src/directives/ripple/_variables.scss:6]: I've thought about that. but the problem is that it won't be documented! if it's written only in ripple `variable.scss`.
directives are not mentioned in [Sass variables page](https://vuetifyjs.com/en/features/sass-variables/#variable-api)

If I keep it in global variable it'll be added to `$vuetify`  object

Comment (KaelWD) [reply packages/vuetify/src/directives/ripple/_variables.scss:6]: Ok remove this file entirely then if it does nothing.

File: packages/vuetify/src/directives/ripple/index.ts (hunk 2 above)
Comment (KaelWD) [reply packages/vuetify/src/directives/ripple/index.ts:119]: Should update the sass value to match the current design.

Comment (KareemDa) [reply packages/vuetify/src/directives/ripple/index.ts:119]: Done  👍

File: packages/vuetify/src/directives/ripple/_variables.scss (hunk 1 above)
Comment (KareemDa) [reply packages/vuetify/src/directives/ripple/_variables.scss:6]: Done  👍

File: packages/vuetify/src/directives/ripple/index.ts (hunk 2 above)
Comment (KareemDa) [reply packages/vuetify/src/directives/ripple/index.ts:119]: Any update on this PR ?

**Review by KaelWD — APPROVED** [review]

**Pull request merged**